profiles/qtile.py

```diff
@@ -0,0 +1,55 @@
+# A desktop environment using "qtile" window manager with common packages.
+
+import archinstall
+
+is_top_level_profile = False
+
+# New way of defining packages for a profile, which is iterable and can be used out side
+# of the profile to get a list of "what packages will be installed".
+__packages__ = [
+	'qtile',
+	'alacritty',
+	'lightdm-gtk-greeter',
+	'lightdm',
+	'dmenu',
+	'pcmanfm',
+	'vim',
```

Comment:
remove vim and nano, these aren't DE specific and don't belong in the desktop profile.